stubs/golang/build.gradle

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.6'
+    }
+}
+
+plugins {
+    id 'com.github.blindpirate.gogradle'
+    id 'java'
+    id 'com.google.protobuf'
+    id 'org.ajoberstar.git-publish'
+}
+
+golang {
+    packagePath = 'github.com/infostellarinc/go-genproto'
```

Comment:
Will this be used as a package name of stubs for Go lang? If so, it should be consistent with GitHub repository to push. 

Since go_package is defined in each proto file, I am not sure if this is needed.
https://github.com/infostellarinc/stellarstation-api/commit/7e8631ce4b4a7e9fda2139f0554bbe120f6713ca

Reply:
Interesting, thanks for the explanation.